pkgs/development/libraries/openldap/default.nix

```
@@ -61,7 +61,8 @@ stdenv.mkDerivation rec {
 
   makeFlags= [
     "CC=${stdenv.cc.targetPrefix}cc"
-    "STRIP="  # Disable install stripping as it breaks cross-compiling. We strip binaries anyway in fixupPhase.
+    "STRIP="       # Disable install stripping as it breaks cross-compiling. We strip binaries anyway in fixupPhase.
+    "STRIP_OPTS="  # Disable install stripping as it breaks cross-compiling. We strip binaries anyway in fixupPhase.
```

```suggestion
    # Disable install stripping as it breaks cross-compiling. We strip binaries anyway in fixupPhase.
    "STRIP="       
    "STRIP_OPTS="
```

Commenting it once is probably sufficient.

---

Done